packages/replay-data/src/bootstrap.ts

```diff
@@ -1,13 +1,33 @@
 /* Copyright 2020-2024 Record Replay Inc. */
 
+import { createRequire } from "module";
 import os from "os";
+import { pathToFileURL } from "url";
 
 import WebSocket from "ws";
 
 /**
- * Hackfixes for some of our dependencies.
+ * Hackfixy JS patching.
  */
+function ignoreMultiMediaImports() {
+  const fileUrl = pathToFileURL(__filename).toString();
```

Comment:
NOTE: We need this, so we can deal with multimedia imports (`import myImage from "myImage.svg";`) from our `devtools` dependency.